src/ORM/DataQuery.php

```diff
@@ -662,9 +662,18 @@ public function having($having)
      */
     public function disjunctiveGroup()
     {
-        return new DataQuery_SubGroup($this, 'OR');
+        // using func_get_args to add a new param while retaining BC
+        // @deprecated - add a new param for CMS 6 - string $clause = 'WHERE'
+        $clause = 'WHERE';
+        $args = func_get_args();
```

Comment:
Also needs a `@param string $clause` in the PHPDoc, so developers can see from within their IDE that this thing can take a string argument.
Same for `conjunctiveGroup` and the `DataQuery_SubGroup` constructor

Reply:
Updated